src/main/java/seedu/java/util/Parser.java

```diff
@@ -0,0 +1,74 @@
+package seedu.java.util;
+
+/**
+ * All static method. Handles input from user.
+ * readCommand(String)
+ * readTask(String)
+ * readTiming(String)
+ * readNum(String)
+ */
+public class Parser {
+    /**
+     * Static method. Returns a Command (inferred instruction)
+     * @param input from UI
+     * @return Command
+     * @throws Exception
+     */
+    public static Command readCommand(String input) throws Exception {
+        return Command.convert(input.split(" ")[0]);
+    }
+
+    /**
+     * Reads user's input and returns a String.
```

Comment:
I think this comment could be more descriptive! For example, why is it called `readTask`? To me, I feel like the method would return a Task, but instead it returns a String

Some pointers you could think of:
- significance of the String returned, where is it used?
- explain more about how "reading" or "interpreting" the task is done